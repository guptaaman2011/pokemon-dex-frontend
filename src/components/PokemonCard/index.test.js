import React from "react";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";
import { PokemonCard } from "..";

describe("PokemonCard", () => {
  it("should render without breaking", () => {
    const wrapper = shallow(<PokemonCard />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
