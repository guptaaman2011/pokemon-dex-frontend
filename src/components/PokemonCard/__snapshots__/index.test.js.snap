// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PokemonCard should render without breaking 1`] = `
<Card
  cover={
    <img
      alt="example"
    />
  }
  hoverable={true}
>
  <Meta />
  <Meta
    title="Height: undefined Weight: undefined"
  />
  <Meta
    title="Abilities: undefined"
  />
</Card>
`;
